test/spec/modules/microadBidAdapter_spec.js

```diff
@@ -266,35 +266,70 @@ describe('microadBidAdapter', () => {
       });
     });
 
-    it('should add Liveramp identity link if it is available in request parameters', () => {
+    it('should not add Liveramp identity link and Audience ID if it is not available in request parameters', () => {
       const bidRequestWithLiveramp = Object.assign({}, bidRequestTemplate, {
```

Comment:
should `bidRequestWithLiveramp` be changed to `bidRequestWithLiveramp`?

Reply:
@lksharma 
Fixed - https://github.com/prebid/Prebid.js/pull/8662/commits/9ed2ded075d730a305e8cb6af186c56c28258f9c